src/client/types.ts

```diff
@@ -326,6 +326,10 @@ export interface RetrieveOptions {
    * Specifies whether to unzip the retrieved zip file. Only applies when `format: metadata`.
    */
   unzip?: boolean;
+  /**
+   * Specifies whether to suppress the <Pre|Post><Retrieve> events
+   */
+  suppressEvents?: boolean;
```

Comment:
Only suppressing Pre and Post retrieve events.  Other events are still fired (like apiVersionRetrieve).